### ISSUE TITLE
Shared folder only displaying one file

### ISSUE THREAD

**Issue by weeman1337** [nextcloud/server#10358]
### Steps to reproduce
1. Create a folder
2. Put at least two files into it
3. Share the folder by link
4. Open the shared link in a private tab/window

### Expected behaviour
All the files are listed.

### Actual behaviour
Online one file is listed.

### Server configuration
**Nextcloud version:** 14.0.0 alpha
**Where did you install Nextcloud from:** git

**Comment by weeman1337** [comment]
The issue is caused by the page size calculation based on the scroll container, happening in `apps/files/js/filelist.js` `pageSize()`. I tracked it down to this commit: https://github.com/nextcloud/server/commit/842583038bb54c2787fb7a2d5ebd84a2fec849fd . It removed the `content-wrapper` DIV and put the `#app-content` scroll container selector into the script instead. An element with the id `app-content` doesn't exist and so the page size is set to 1 → displaying one file.

**Comment by skjnldsv** [comment]
Yep, we maybe rushed this pr a bit too fast. But I'd rather fix things quickly now! :)
Thanks for finding this out, we need to adapt the guest page to the new structure!
